apps/ethereum_jsonrpc/lib/ethereum_jsonrpc/contract.ex

```diff
@@ -36,22 +36,29 @@ defmodule EthereumJSONRPC.Contract do
 
     requests_with_index = Enum.with_index(requests)
 
-    indexed_responses =
-      requests_with_index
+    prepared = requests_with_index
       |> Enum.map(fn {%{contract_address: contract_address, function_name: function_name, args: args} = request, index} ->
         functions[function_name]
         |> Encoder.encode_function_call(args)
         |> eth_call_request(contract_address, index, Map.get(request, :block_number))
       end)
+    
+    IO.inspect(prepared)
```

Comment:
Should we keep this? Maybe adding with `Logger` if make sense to have in the production log?

Reply:
I'll check if logger has different levels. Otherwise they'll all have to go